manifests/g/GoXLR-on-Linux/GoXLR-Utility/1.0.3/GoXLR-on-Linux.GoXLR-Utility.locale.en-GB.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: GoXLR-on-Linux.GoXLR-Utility
+PackageVersion: 1.0.3
+PackageLocale: en-GB
+Publisher: GoXLR on Linux

```

```suggestion
Publisher: The GoXLR on Linux Team
```
This should match control panel exactly